CONTRIBUTING.md

```diff
@@ -189,11 +189,23 @@ with `./mvnw -Dquickly`.
 
 ##### `OutOfMemoryError` while importing
 
-After creating an IDEA project, the first import will most likely fail with an `OutOfMemoryError`.
+After creating an IDEA project, the first import might fail with an `OutOfMemoryError`,
+as the size of the project requires more memory than the IDEA default settings allow.
 
 To fix that, open the _Preferences_ window (or _Settings_ depending on your edition),
 then navigate to _Build, Execution, Deployment_ > _Build Tools_ > _Maven_ > _Importing_.
-In _VM options for importer_, raise the heap to at least 2 GB, e.g. `-Xmx2g -Xms2g`.
+In _VM options for importer_, raise the heap to at least 2 GB; some people reported
+needing more, e.g. `-Xmx8g`.
+
+In some most recent IDEA versions this won't work because of open issue [IDEA-250160](https://youtrack.jetbrains.com/issue/IDEA-250160):
+it disregards the _VM options for importer_ settings.
```

Comment:
```suggestion
```

Reply:
I'd keep this, it summarizes the problem.

Reply:
Isn't that covered by "this might not work because _VM options for importer_ get ignored when `.mvn/jdk.config` is present" above?

Reply:
I might be wrong but I had the impression the bug is more extensive than that.